main.js

```diff
@@ -0,0 +1,20 @@
+let el = document.querySelector("nav li");
+el.remove();
+
+let ele = document.querySelector("aside span");
+ele.remove();
+
+const h3tags = document.querySelectorAll(".posts article");
+
+for (let el of h3tags) {
+  if (el.textContent.includes("You -- yes you!")) {
+    el.remove();
+  }
+}
+
```

Comment:
really interesting way to do this one, nice job!

here's how I did it in case you're curious:
`h3tags[h3tags.length - 1].remove()` <-- grabbing the last element in the `h3tags` node list and deleting it.